source/_components/stream.markdown

```diff
@@ -25,3 +25,21 @@ To enable this component, add the following lines to your `configuration.yaml` f
 # Example configuration.yaml entry
 stream:
 ```
+
+## {% linkable_title Troubleshooting %}
```

Comment:
Is this needed for hassbian too?

Reply:
I don't believe it is based on the discussion here: https://github.com/home-assistant/home-assistant.io/pull/8838#issuecomment-472079802